packages/types/src/primitive/Null.ts

```diff
@@ -18,6 +22,13 @@ export default class Null implements Codec {
     return 0;
   }
 
+  /**
+   * @description returns a hash of the contents
+   */
+  public get hash (): IHash {
+    return createType('Hash', blake2AsU8a(this.toU8a(), 256));
```

Comment:
hmm, it's sometimes `createType('Hash',)`, sometimes `new U8a(...)`. For consistency I would just use createType everywhere

Reply:
Can't use `createType` in codec without running into the issues of `extending from null`

i.e. in U8a -

- U8aFixed extends U8a
- U8a imports createType
- createType imports U8aFixed
- circular dep, all hell breaks loose :)

Reply:
ah, f, I see